### ISSUE TITLE
Add support for FDSStick RAW flux images

### ISSUE THREAD

**Issue by FakeShemp** [aaru-dps/Aaru#628]
### Check and fill as appropriate:
* [X] Support for a new media image format...

## Detailed Description

A Famicom Disk System (FDS) drive emulator and dumper.

## Known documentation

http://3dscapture.com/fdsstick/

**Comment by eadmaster** [comment]
[FDS images](https://wiki.nesdev.org/w/index.php/FDS_disk_format) write support is included with this?

**Comment by claunia** [comment]
@eadmaster can you elaborate your question please?

**Comment by eadmaster** [comment]
does this include support to read and modify the files inside FDS images?

**Comment by FakeShemp** [comment]
@eadmaster No, this is just for the raw flux images created by FDSStick. Feel free to create other issues for any additional formats you would like to see supported.
